src/System Application/App/AI/src/Azure OpenAI/AOAIAuthorization.Codeunit.al

```diff
@@ -95,6 +73,16 @@ codeunit 7767 "AOAI Authorization"
         ApiKey := NewApiKey;
         ManagedResourceDeployment := NewManagedResourceDeployment;
     end;
+#endif
+
+    [NonDebuggable]
```

Comment:
Remove the ApiKey/AOAIAccountName from the original function.
This codeunit is internal, so we can make those changes as they are non-breaking. I'd like to reduce the redundancy in the implementation codeunit.

Just point the facade's API to the right procedure now and it might need a pragma there.